contracts/Fees.sol

```diff
@@ -43,7 +43,12 @@ abstract contract Fees is IFees, Owned {
                 protocolWithdrawFee = uint16(updatedProtocolFees & 0xFFF);
```

Comment:
can this line revert?

Reply:
it shouldnt, you could write some fuzz tests though on different protocolFee values returned though